src/fvm_multicell.hpp

```diff
@@ -501,6 +525,17 @@ void fvm_multicell<Backend>::initialize(
     // initialize storage from total compartment count
     current_ = array(ncomp, 0);
     voltage_ = array(ncomp, resting_potential_);
+    cv_to_cell_ = iarray(ncomp, 0);
+    time_ = array(ncell_, 0);
+    time_to_ = array(ncell_, 0);
+
+    // initialize cv_to_cell_ values from compartment partition
+    std::vector<size_type> cv_to_cell_tmp(ncomp);
+    for (size_type i = 0; i<ncell_; ++i) {
+        auto comps = cell_comp_part[i];
+        std::fill(cv_to_cell_tmp.begin()+comps.first, cv_to_cell_tmp.begin()+comps.second, i);
```

Comment:
Elsewhere, this same operation is done with the construction `util::fill(util::subrange_view(vec, span), value)`, which IMO is cleaner and should be used here too.